--- libimagequant-2.17.0/Makefile.orig	2021-11-15 00:57:01.000000000 +0100
+++ libimagequant-2.17.0/Makefile	2022-03-16 21:05:11.342090757 +0100
@@ -64,8 +64,8 @@ libimagequant.dylib: $(SHAREDOBJS)
 
 $(OBJS): $(wildcard *.h) config.mk
 
-$(JNILIB): $(JAVAHEADERS) $(STATICLIB) org/pngquant/PngQuant.c
-	$(CC) -g $(CFLAGS) $(LDFLAGS) $(JAVAINCLUDE) -shared -o $@ org/pngquant/PngQuant.c $(STATICLIB)
+$(JNILIB): $(JAVAHEADERS) $(SHAREDLIB) org/pngquant/PngQuant.c
+	$(CC) -g $(CFLAGS) $(LDFLAGS) $(JAVAINCLUDE) -shared -o $@ org/pngquant/PngQuant.c $(SHAREDLIB)
 
 $(JNIDLL) $(JNIDLLIMP): $(JAVAHEADERS) $(OBJS) org/pngquant/PngQuant.c
 	$(CC) -fPIC -shared -I. $(JAVAINCLUDE) -o $(JNIDLL) $^ $(LDFLAGS) -Wl,--out-implib,$(JNIDLLIMP),--output-def,$(JNIDLLDEF)
@@ -134,7 +134,7 @@ ifeq ($(filter %clean %distclean, $(MAKE
 endif
 
 $(PKGCONFIG): config.mk
-	sed 's|@PREFIX@|$(PREFIX)|;s|@VERSION@|$(VERSION)|' < imagequant.pc.in > $(PKGCONFIG)
+	sed 's|@PREFIX@|$(PREFIX)|;s|@LIBDIR@|$(LIBDIR)|;s|@VERSION@|$(VERSION)|' < imagequant.pc.in > $(PKGCONFIG)
 
 .PHONY: all static shared clean dist distclean dll java cargo
 .DELETE_ON_ERROR:
--- libimagequant-2.17.0/imagequant.pc.in.orig	2022-03-16 21:03:51.702522202 +0100
+++ libimagequant-2.17.0/imagequant.pc.in	2022-03-16 21:05:58.435168965 +0100
@@ -1,6 +1,6 @@
 prefix=@PREFIX@
 includedir=${prefix}/include
-libdir=${prefix}/lib
+libdir=@LIBDIR@
 
 Name: imagequant
 Description: Small, portable C library for high-quality conversion of RGBA images to 8-bit indexed-color (palette) images.
