--- libimagequant-2.11.10/Makefile.orig	2018-03-10 13:31:19.000000000 +0100
+++ libimagequant-2.11.10/Makefile	2018-05-27 22:00:19.433276005 +0200
@@ -51,6 +51,7 @@
 	$(CC) -shared -Wl,-soname,$(SHAREDLIB).$(SOVER) -o $(SHAREDLIB).$(SOVER) $^ $(LDFLAGS)
 	ln -fs $(SHAREDLIB).$(SOVER) $(SHAREDLIB)
 	sed -i "s#^prefix=.*#prefix=$(PREFIX)#" $(PKGCONFIG)
+	sed -i "s#^libdir=.*#libdir=$(LIBDIR)#" $(PKGCONFIG)
 	sed -i "s#^Version:.*#Version: $(VERSION)#" $(PKGCONFIG)
 
 libimagequant.dylib: $(SHAREDOBJS)
@@ -59,8 +60,8 @@
 
 $(OBJS): $(wildcard *.h) config.mk
 
-$(JNILIB): $(JAVAHEADERS) $(STATICLIB) org/pngquant/PngQuant.c
-	$(CC) -g $(CFLAGS) $(LDFLAGS) $(JAVAINCLUDE) -shared -o $@ $(STATICLIB) org/pngquant/PngQuant.c
+$(JNILIB): $(JAVAHEADERS) $(SHAREDLIB) org/pngquant/PngQuant.c
+	$(CC) -g $(CFLAGS) $(LDFLAGS) $(JAVAINCLUDE) -shared -o $@ $(SHAREDLIB) org/pngquant/PngQuant.c
 
 $(JNIDLL) $(JNIDLLIMP): $(JAVAHEADERS) $(OBJS) org/pngquant/PngQuant.c
 	$(CC) -fPIC -shared -I. $(JAVAINCLUDE) -o $(JNIDLL) $^ $(LDFLAGS) -Wl,--out-implib,$(JNIDLLIMP),--output-def,$(JNIDLLDEF)
